metrics-jmx/src/main/java/com/codahale/metrics/jmx/JmxReporter.java

```diff
@@ -544,6 +544,7 @@ public void onGaugeAdded(String name, Gauge<?> gauge) {
             } catch (JMException e) {
                 LOGGER.warn("Unable to register gauge", e);
             }
+            return null;
```

Comment:
Why don't we return the `Gauge<?>` which has been passed into the method when it was added?